src/main/java/seedu/address/seplendidui/PartnerCourseItem.java

```diff
@@ -33,5 +35,6 @@ public PartnerCourseItem(PartnerCourse partnerCourse, int displayedIndex) {
         id.setText(displayedIndex + ".");
         partnerName.setText(partnerCourse.getPartnerName().toString());
         partnerCode.setText(partnerCourse.getPartnerCode().toString());
+        partnerUniversity.setText(partnerCourse.getPartnerUniversity().getUniversityName().toString());
```

Comment:
I think you can straightaway call toString() on university

Reply:
i think cannot